fili-sql/src/main/java/com/yahoo/bard/webservice/sql/DruidQueryToSqlConverter.java

```diff
@@ -839,6 +848,7 @@ private void sqlAggregationHelper(
         aggToSqlType.put("longMin", SqlTypeName.BIGINT);
         return druidQuery.getAggregations()
                 .stream()
+                .filter(aggregation -> !aggregation.getType().equals("thetaSketch"))
```

Comment:
ThetaSketchAggregation.AGGREGATION_TYPE as the magic string here to support compatibility if things ever change.